tests/system/fim/scenarios/204_whodata_linux_noaudit/launch_test_scenario.yml

```diff
@@ -8,45 +8,61 @@
 
     tasks:
       - block:
+        - name: Auditd Uninstall
+          become: true
+          import_tasks: "actions/uninstall_audit.yml"
+
         - name: Launching prepare tasks
           import_tasks: "../../common_tasks/prepare.yml"
 
-        - name: Check for disabled auditd. Check for realtime running instead.
-          become: true
-          import_tasks: "tests/tests.yml"
+        - name: Wait until whodata switch to realtime message.
+          become: True
+          wait_for:
+            path: "{{ linux_ossec_folder_path }}/logs/ossec.log"
+            search_regex: 'Who-data engine could not start. Switching who-data to real-time.'
+          when:
+            - inventory_hostname in groups['linuxagents']
```

Comment:
Please consider adding a custom timeout (default is 300s)

Reply:
The timeout is now 60 seconds